pyproject.toml

```diff
@@ -41,6 +41,7 @@ covcheck = [
 ]
 test = [
     "pytest>=8.1.0",
+    "vectorplot",
```

Comment:
an issue with this approach is that it only works with uv's high level API (`uv run`, `uv sync`), and will *silently* fail for other standard compliant interfaces (including `pip install --group` and `uv pip install --group`), because there's a different package named `vectorplot` on PyPI.

Reply:
(resolved by instead including `vectorplot` in a uv-only group)